10up/classifai
# Add better error handling for manual scanning of alt text or image tags

### Description of the Change

Make sure our requests to generate alt text or image tags return a WP_Error object if something goes wrong. Currently we are returning those in a few instances but not in every instance.

In our javascript that makes these requests, if the response fails, check to see if we have error details and if so, show that error message. If we don't have error details, output a default error message. Also, make sure the spinner is removed so it doesn't appear like things are still processing.

### Alternate Designs

None

### Benefits

If a request fails, we provide feedback to the user about what went wrong.

### Possible Drawbacks

None

### Verification Process

1. Setup the plugin
2. Disable either the Automatically Caption Images or Automatically Tag Images options (or both)
3. Select an existing image in the media library or upload a new one
4. With the image selected, in the media modal that opens, click the button that corresponds to the setting disabled above. Ensure an error message is shown and the spinner goes away

![Screen Shot 2020-10-19 at 12 32 37 PM](https://user-images.githubusercontent.com/916738/96497379-d4d91680-1207-11eb-8b5f-fc6a7cb781e4.png)

### Checklist:


### Applicable Issues

#230
